README.md

```diff
@@ -1,15 +1,111 @@
+
+And during the COVID-19  pandemic, we can use this game to keep ourselves entertained while maintaining our social circle.  What’s more, this game will keep you engaged and help reduce stress. This time you neither need to  fret over saying UNO before you drop your penultimate card nor wait until the deck is shuffled.
+
+By  this project, we intend to develop a UNO game engine that can make choices xadhering  to the rules of the game. 
+This game is similar to the well known UNO game but following modifications are made to deliver this game efficiently as a single player game with a computer opponent.
+
+* UNO Express card game has 56 cards, we removed the 'Reverse' action card as it doesn't make much of a sense in a two player game.So,
+  the total card count is 52.
+* Human player always starts the game first.
+* When the Draw two action card is played, the next player automatically loses his turn, gets two cards added to his current set 
+  set of cards and the current player can choose the color to continue the game (same as a normal UNO game).
+* When the draw one action card is played, the next player by default loses his turn and a card will be added to his set of cards.
+
+
+## Group Information
+
+**Organization:** Carleton Unviersity (SYSC 5709)
+
+### Authors:
+
+**1. Mia Wei**
+ - GitHub: MiaYWei
+
+**2. Krithiga Subramaniam Ravichandran**
+
+ - GitHub: krithigasr
+
+**3. Hongbo Pang**
+
+ - GitHub: HongBo713
+
+**4. Aditi Bhardwaj**
+
+ - GitHub: aditibhardwaj98
+
+## Brief Description
+Following are the rules with which the game has been designed
+
+1. There are 52 cards in total - 40 normal cards, 12 action cards.
+2. These are 10 cards in each color - red,green,yellow and blue and action cards are - 4 Draw One cards, 4 Skip cards, 2 Wild cards,
+   2 Wild Draw two cards.
+3. To be declared as the winner of the game, you have to be the first player to exhaust all the cards assigned to you.
+4. During the start of the game, 5 random cards are assigned to each player - human player and the computer, in this case.
+5. One card is placed on the table with which you have to start matching your cards.
+6. To keep things simple, the human player always starts the game first.
+7. These are the keyboard inputs with which human can play the game
+   * To request a card from the Draw pile, Press N or n.
+   * To end turn, Press E or e.
+   * To discard a normal card - Say you want to discard(RED,3), Press R3.
+   * To discard action cards, 
+     * Skip cards- Combine the first letter of the color with 'S', so if you want to discard(RED,SKIP), Press RS.
+     * Draw one cards- Combine the first letter of the color with 'O', so if you want to discard(RED,DRAW-ONE), Press R0.
+     * Wild cards- Press AW.
+     * Wild cards- Press AT.
+     (A above stands for Action cards, W stands for Wild, T stands for draw Two)
```

Comment:
Wild and Wild Draw Two

![image](https://user-images.githubusercontent.com/33041048/102626521-e9a52e00-4114-11eb-8909-a44cbdfa0a63.png)